ghpghx/models.py

```diff
@@ -0,0 +1,300 @@
+        help_text="Peak heating heat pump thermal load served [ton]")
+    peak_cooling_heatpump_thermal_ton = models.FloatField(null=True, blank=True, 
+        help_text="Peak cooling heat pump thermal load served [ton]")
+    peak_combined_heatpump_thermal_ton = models.FloatField(null=True, blank=True, 
+        help_text="Peak combined heating/cooling heat pump thermal load served [ton]")
+    max_eft_f = models.FloatField(null=True, blank=True, 
+        help_text="Maximum entering fluid temperature (to heat pump, from ghx) throughout the GHPGHX simulation period")
+    min_eft_f = models.FloatField(null=True, blank=True, 
+        help_text="Minimum entering fluid temperature (to heat pump, from ghx) throughout the GHPGHX simulation period")
+    heating_cop_avg = models.FloatField(null=True, blank=True, 
+        help_text="Average heating heatpump system coefficient of performance (COP) (includes ghx pump allocation)")
+    cooling_cop_avg = models.FloatField(null=True, blank=True, 
+        help_text="Average cooling heatpump system coefficient of performance (COP) (includes ghx pump allocation)")
+    solved_eft_error_f = models.FloatField(null=True, blank=True,
+        help_text="Error between the solved GHPGHX system EFT and the max or min limit for EFT")
+
+
+class ModelManager(object):
+
+    def __init__(self):
+        self.ghpghxM = None
+
+    @staticmethod
+    def make_response(ghp_uuid):
+        """
+        Reconstruct response dictionary from postgres tables (django models).
+        # TODO Make sure the NOTE below is not relevant for ghpghx because it uses primary_key=True for ghp_uuid
+        NOTE: postgres column type UUID is not JSON serializable. Work-around is removing those columns and then
+              adding back into outputs->Scenario as string.
+        :param ghp_uuid:
+        :return: nested dictionary matching nested_output_definitions
+        """
+
+        resp = dict()
+        resp["ghp_uuid"] = str(ghp_uuid)
+        resp["outputs"] = dict()
+        resp["inputs"] = dict()
+        resp["messages"] = dict()
+
+        try:
+            ghpghx_model = GHPGHXModel.objects.get(ghp_uuid=ghp_uuid)
+        except Exception as e:
+            if isinstance(e, models.ObjectDoesNotExist):
+                resp['messages']['error'] = (
+                    "ghp_uuid {} not in database. "
+                    "You may have hit the results endpoint too quickly after POST'ing scenario, "
+                    "you may have a typo in your ghp_uuid, or the scenario was deleted.").format(ghp_uuid)
+                resp['status'] = 'error'
+                return resp
+            else:
+                raise Exception
+        #ghpghx_data = remove_ids(model_to_dict(ghpghx_model))
+        ghpghx_data = model_to_dict(ghpghx_model)
+
+        # Listed here to separate outputs from the model, so need to keep this update with added outputs to model
```

Comment:
We would not have to keep the `output_fields` updated if we split the GHPGHXModel into its inputs and outputs (like in v2 job/models). Should we split the model into two?

Reply:
This is a good idea and should be done. But, I'm going to punt on this until I convert GHP to V2 when I'm in the process of converting all the other thermal technologies over.

Reply:
There may be a theme in my remaining responses too, that I'll try to address everything that is required now, but I may punt on "nice to haves" because of time crunch now. We are pushing off the webtool public release until Q1 FY22, and I'll have more time to address the nice-to-haves then as well.

Reply:
I ended up splitting these up per your suggestion.